ci-operator/config/rh-messaging-qe/claire/rh-messaging-qe-claire-lpt__amq-broker-ocp4.14-lp-rosa-classic.yaml

```diff
@@ -0,0 +1,67 @@
+              {"step": "amq-broker-install-and-tests", "failure_type": "test_failure", "classification": "Test Failure", "jira_project": "ENTMQBR", "group": {"name": "lp-tests", "priority": 1}, "jira_additional_labels": ["!default","interop-tests"]},
+              {"step": "aws-provision*", "failure_type": "all", "classification": "Infrastructure Provisioning - AWS", "group": {"name": "cluster", "priority": 1}, "jira_additional_labels": ["!default"]},
+              {"step": "aws-deprovision*", "failure_type": "all", "classification": "Infrastructure Deprovisioning - AWS", "group": {"name": "cluster", "priority": 2}, "jira_additional_labels": ["!default"]},
+              {"step": "cucushift-*", "failure_type": "all", "classification": "Health Check - Cluster", "group": {"name": "lp-tests", "priority": 2}, "jira_additional_labels":["!default"]},
+              {"step": "gather-*", "failure_type": "pod_failure", "classification": "Must Gather - Cluster", "group": {"name": "cluster", "priority": 2}, "jira_additional_labels": ["!default"]},
+              {"step": "gather-*", "failure_type": "test_failure", "classification": "NONE", "jira_project": "NONE", "ignore": "true"},
+              {"step": "openshift-*", "failure_type": "all", "classification": "Other", "group": {"name": "cluster", "priority": 2}, "jira_additional_labels": ["!default"]},
+              {"step": "rosa-cluster-deprovision", "failure_type": "all", "classification": "Infrastructure Deprovisioning - Cluster", "group": {"name": "cluster", "priority": 2}, "jira_additional_labels": ["!default"]},
+              {"step": "rosa-cluster-provision", "failure_type": "all", "classification": "Infrastructure Provisioning - Cluster", "group": {"name": "cluster", "priority": 1}, "jira_additional_labels": ["!default"]},
+              {"step": "rosa-cluster-wait*", "failure_type": "all", "classification": "Infrastructure Provisioning - Rosa Cluster Operators", "group": {"name": "cluster", "priority": 2}, "jira_additional_labels": ["!default"]},
+              {"step": "rosa-sts-account-roles-create", "failure_type": "all", "classification": "Account Roles Creation - AWS", "group": {"name": "cluster", "priority": 2}, "jira_additional_labels": ["!default"]},
+              {"step": "rosa-sts-account-roles-delete", "failure_type": "all", "classification": "Account Roles Deletion - AWS", "group": {"name": "cluster", "priority": 3}, "jira_additional_labels": ["!default"]},
+              {"step": "rosa-conf-idp-htpasswd", "failure_type": "all", "classification": "Admin Access - Cluster", "group": {"name": "cluster", "priority": 2}, "jira_additional_labels": ["!default"]}
+            ]
+        }
+      FIREWATCH_DEFAULT_JIRA_ADDITIONAL_LABELS: '["4.14-lp","rosa-lp","amq-broker-lp"]'
```

Comment:
Topic: changing the label `rosa-lp` to `rosa-classic-lp`

We're using `rosa-hypershift-lp` [here](https://github.com/openshift/release/blob/master/ci-operator/config/rhpit/interop-tests/rhpit-interop-tests-main__openshift-pipelines-hypershift-4.14-lp-rosa-hypershift.yaml#L57)

Because of that maybe we can just assume rosa-lp will mean rosa classic and then we know rosa-hypershift-lp is for rosa hypershift. I'm not sure if this is the intention or not, what are your thoughts?

Reply:
REF - https://github.com/openshift/release/pull/46606

Reply:
Okay @ascerra, I agree with your point to use `rosa-classic-lp` instead of using `rosa-lp` label for more clarity.
Previously, we used same `rosa-lp` label for the rosa-classic scenarios.

Reply:
@ascerra , please let me know will I change `rosa-classic-lp` label for all rosa classic scenarios?

Reply:
If you agree with this then I believe changing this label for all rosa classic scenarios will be a good thing.

@oharan2 do you see any reason or feel a certain way about why we shouldn't do this?

Reply:
Added it for other ROSA Classic onboarded scenarios as well : [#47710 ](https://github.com/openshift/release/pull/47710)